Gemfile

```
@@ -15,3 +15,4 @@ gem 'rubocop-rake', require: false
 gem 'rubocop-rspec', require: false
 gem 'vcr'
 gem 'webmock'
+gem 'pry'
```

RuboCop (1.75.5)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pry` should appear before `webmock`.

(Bundler/OrderedGems)